### ISSUE TITLE
[Refactoring]

### ISSUE THREAD

**Issue by simaolemos** [SaraVieira/stop-microspending#19]
a component should call a vuex action and not a mutation using the nextTick

component (call) -> action (call) -> mutation and the mutation change the store

**Comment by SaraVieira** [comment]
Oh no, but the  it will become redux :(

**Comment by simaolemos** [comment]
yes will becode something like redux, the vuex is basicly the redux to vue.

**Comment by SaraVieira** [comment]
I know but I dislike the complexity :(

Do you have any reasons to switch this?

**Comment by simaolemos** [comment]
The unique reason is that if you want to use the vuex you should follow the vuex best practices

**Comment by SaraVieira** [comment]
Damn it you are right :(
